tests/testthat/test-utils.R

```diff
@@ -223,6 +223,36 @@ test_that("average_biological_replicates_dt works as expected", {
   expect_equal(dim(avg_metrics_data2), c(40, 44))
   expect_equal(sum(grepl("_sd", names(avg_metrics_data2))), 15)
   expect_true("count" %in% names(avg_metrics_data2))
+  
+  # protection against regression
+  # (GDR-2591, fit_type correctly recognized in wide and long format)
```

Comment:
```suggestion
  # fit_type correctly recognized in wide and long format
```

(we use `GDR-xxxx` for `ToDos` 😄 , but the comment about the type of test is a great idea)